datahub-kubernetes/datahub/charts/datahub-frontend/templates/service.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: {{ include "datahub-frontend.fullname" . }}
   labels:
     {{- include "datahub-frontend.labels" . | nindent 4 }}
+  {{- with .Values.service.annotations }}
```

Comment:
Can we add to GMS service.yaml as well?

Reply:
@dexter-mh-lee definitely I will send another PR for that once this is approved.

Reply:
@dexter-mh-lee pushed another commit for gms service